sql/core/src/test/resources/sql-tests/inputs/operator.sql

```diff
@@ -32,3 +32,11 @@ select 1 - 2;
 select 2 * 5;
 select 5 % 3;
 select pmod(-7, 3);
+
+-- check operator precedence
```

Comment:
As arithmetic.sql is renamed to operator.sql and operator.sql.out is added, shall we remove arithmetic.sql.out?